redcap_bridge/utils.py

```diff
@@ -209,18 +208,23 @@ def radio_to_dict(redcap_field_dict):
     return temp_elab_dict
 
 
-def checkbox_to_dict(redcap_field_dict):
+def checkbox_to_multiselect_dict(redcap_field_dict):
     assert redcap_field_dict["Field Type"] == "checkbox"
     redcap_choice_str = redcap_field_dict["Choices, Calculations, OR Slider Labels"]
     redcap_annotation_str = redcap_field_dict["Field Annotation"]
     choice_labels, default_choice_label = parse_choices(redcap_choice_str, redcap_annotation_str)
     temp_elab_dict = {
         redcap_field_dict['Field Label']: {
-            "type": "checkbox",
+            "type": "select",
             "value": default_choice_label,
-            "options": choice_labels
+            "options": choice_labels,
+            "allow_multi_values": True
         },
     }
+
+    for key in temp_elab_dict[redcap_field_dict['Field Label']]:
+        json.dumps(temp_elab_dict[redcap_field_dict['Field Label']])
```

Comment:
I did this because in python True takes a capital T while in json true takes a lowercase t. This is the only way I found to do the translation without getting an error